doc/api/process.md

```diff
@@ -1775,6 +1784,317 @@ Will generate an object similar to:
   tz: '2016b' }
 ```
 
+## Performance Timing API
+<!-- YAML
+added: REPLACEME
+-->
+
+The Performance Timing API provides an implementation of the
+[W3C Performance Timeline][] specification. The purpose of the API
```

Comment:
Intentionally started with the first version, will be updating to the latest. The plan is to update incrementally.